src/Snapshooter.Xunit/SnapshotExtension.cs

```diff
@@ -95,7 +96,23 @@ public static class SnapshotExtension
                                  SnapshotNameExtension snapshotNameExtension,
                                  Func<MatchOptions, MatchOptions> matchOptions = null)
         {
-            Snapshot.Match(currentResult, snapshotName, snapshotNameExtension, matchOptions);
+            Snapshot.Match(RemoveUnwantedWrappers(currentResult), snapshotName, snapshotNameExtension, matchOptions);
+        }
+
+        private static object RemoveUnwantedWrappers(object objectToRemoveWrappers)
+        {
+            Type resultType = objectToRemoveWrappers.GetType();
```

Comment:
Extract it to RemoveFluentAssertionWrapper method, just in case we have other unwanted wrappers...